app/backend/store/inmemory_image_store.py

```diff
@@ -0,0 +1,15 @@
+import random
+
+class InMemoryImageStore:
+    """
+        InMemory store to store images.
+    """
+    images = []
+
+    def store_image(self, image_content: bytes, image_description: str):
+        """ Store an image and description."""
+        self.images.append((image_content, image_description))
+
+    def get_random_image(self):
```

Comment:
=> add get_image_description(self, image_id)

for api logic:
=> verify_guess(image_id, description)